### ISSUE TITLE
CTR DRBG: noncelen < 128 not accepted

### ISSUE THREAD

**Issue by smuellerDD** [usnistgov/ACVP#239]
When submitting a CTR DRBG register request for AES-128, the server rejects noncelen < 128:

Server response:
```
{
  "acv_version" : "0.4",
  "error" : "Property value(64) for (nonce) in Algorithm (ctrDRBG (null)) is not valid. Validation: (Number Between (128) and (34359738368) modulus 8)"
}
```

Note, SP800-90A defines the nonce as half the entropy string length. For AES-128, the entropy string length is at least 128 bits.

**Comment by atvassilev** [comment]
This is an issue for the spec and the implementation on the server.

**Comment by atvassilev** [comment]
The spec error has been fixed with PR #243. The web server fix is still pending.
